### ISSUE TITLE
A11y_mssql for VSCode_ Query History_Keyboard : When we perform action on “More actions” button, button will get invisible.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/vscode-mssql#1686]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

**GitHubTags:** #A11y_mssql for VSCode Object Explorer; #A11ySev2; #A11yTCS; #A11yMAS; #DesktopApp;#MAS2.4.7;#Win10;#mssql for VSCode;#Keyboard;#FTP; #Benchmark;
 
**Environment Details:**
Application Name: mssql for VSCode

Screen Reader: NVDA, JAWS 2020

**Repro Steps:** 

1. Open Visual studio code 
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server  button
4. Click on Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Click on “New Query” wirte any valid query and click on "Execute query button"
7. in the left panel Query History window will appears

**Actual:**

When we perform action on “More actions” button, button will get invisible.
**Same issue is also observed when we perform action on "Pause Query Hisory capture" button.**

**Expected:**

Button Should be visible and active When we perform action on “More actions” button. Also keyboard focus should also on "More action" button. It should not invisible after performed  action.

**User Impact:**

Keyboard user will get confuse and not be able to get focusable element information.

**Attachment link for Reference:**
[1686_A11y_mssql for VSCode_ Query History_Keyboard.zip](https://github.com/microsoft/vscode-mssql/files/4687039/1686_A11y_mssql.for.VSCode_.Query.History_Keyboard.zip)

**Comment by abist** [comment]
If the action is performed via the keyboard, the focus is set. However, when you click the button, the keyboard focus won't be there by design.

**Comment by hannahqin** [comment]
Updated in conformance documentation.

**Comment by hannahqin** [comment]
Updated in conformance doc

**Comment by Roopesh-T** [comment]
As the focus is moving to the expend item with the one down arrow key after expanding. closing the bug. and the selection of the more options is visible as a greyed control. 
![image](https://user-images.githubusercontent.com/86601127/218413590-c4e40753-4f54-46e9-b6ce-b9617ac575a7.png)
![1686_Closed](https://user-images.githubusercontent.com/86601127/218414227-294959bb-db7e-4555-8c49-cfe4b8f46587.gif)
